internal/commands/import_test.go

```diff
@@ -0,0 +1,59 @@
+//go:build !integration
+
+package commands
+
+import (
+	"testing"
+
+	errorConstants "github.com/checkmarx/ast-cli/internal/constants/errors"
+	"github.com/checkmarx/ast-cli/internal/wrappers/mock"
+	"gotest.tools/assert"
+)
+
+func TestImport_ImportSarifFileWithCorrectFlags_CreateImportSuccessfully(t *testing.T) {
+	execCmdNilAssertion(t, "import", "--project-name", "my-project", "--import-file-path", "my-path.sarif")
+}
+
+func TestImport_ImportSarifFileProjectDoesntExist_CreateImportWithProvidedNewNameSuccessfully(t *testing.T) {
+	execCmdNilAssertion(t, "import", "--project-name", "MOCK-PROJECT-NOT-EXIST", "--import-file-path", "my-path.sarif")
+}
+
+func TestImport_ImportSarifFileMissingImportFilePath_CreateImportReturnsErrorWithCorrectMessage(t *testing.T) {
+	err := execCmdNotNilAssertion(t, "import", "--project-name", "my-project", "--import-file-path", "")
+	assert.Assert(t, err.Error() == errorConstants.ImportFilePathIsRequired)
+}
+
+func TestImport_ImportSarifFileMissingImportProjectName_CreateImportReturnsErrorWithCorrectMessage(t *testing.T) {
+	err := execCmdNotNilAssertion(t, "import", "--import-file-path", "my-path.zip")
+	assert.Assert(t, err.Error() == errorConstants.ProjectNameIsRequired)
+}
+
+func TestImport_ImportSarifFileProjectNameNotProvided_CreateImportWithProvidedNewNameSuccessfully(t *testing.T) {
+	err := execCmdNotNilAssertion(t, "import", "--project-name", "", "--import-file-path", "my-path.sarif")
+	assert.Assert(t, err.Error() == errorConstants.ProjectNameIsRequired)
+}
+
+func TestImport_ImportSarifFileUnacceptedFileExtension_CreateImportReturnsErrorWithCorrectMessage(t *testing.T) {
+	err := execCmdNotNilAssertion(t, "import", "--project-name", "MOCK-PROJECT-NOT-EXIST", "--import-file-path", "my-path.txt")
+	assert.Assert(t, err.Error() == errorConstants.SarifInvalidFileExtension)
+}
+
+func TestImport_ImportSarifFileMissingExtension_CreateImportReturnsErrorWithCorrectMessage(t *testing.T) {
+	err := execCmdNotNilAssertion(t, "import", "--project-name", "MOCK-PROJECT-NOT-EXIST", "--import-file-path", "some/path/no/extension/my-path")
+	assert.Assert(t, err.Error() == errorConstants.SarifInvalidFileExtension)
+}
+
+func TestImporFileFunction_FakeUnauthorizedHttpStatusCode_ReturnRelevantError(t *testing.T) {
+	err := importFile(mock.FakeUnauthorized401, "importFilePath", &mock.UploadsMockWrapper{}, &mock.ByorMockWrapper{})
```

Comment:
Because it's not a command, but testing of the importFile function. These importFile tests are meant to verify that the http errors are mapped to user friendly errors.